docs/_functions/domain/AZURE_ALIAS.md

```diff
@@ -0,0 +1,40 @@
+---
+name: AZURE_ALIAS
+parameters:
+  - name
+  - type
+  - target
+  - modifiers ...
+---
+
+AZURE_ALIAS is a Azure specific virtual record type that points a record at either another record or an Azure entity. It is analogous to a CNAME, but is usually resolved at request-time and served as an A record. Unlike CNAMEs, ALIAS records can be used at the zone apex (`@`)
```

Comment:
Please wrap markdown text around 80 chars.  This way future diffs are smaller.

Reply:
Updated the text